docs/source/installation/guide_complete/index.rst

```diff
@@ -0,0 +1,363 @@
+
+        #. Install `pip <https://pip.pypa.io/en/stable/installation/>`_.
+        #. Install ``aiida-core``:
+
+           .. code-block:: console
+
+               pip install aiida-core
+
+    .. tab-item:: conda
+
+        Installing ``aiida-core`` using Conda.
+
+        #. Install `conda <https://conda.io/projects/conda/en/latest/user-guide/install/index.html>`_.
+
+        #. Create an environment and install ``aiida-core``:
+
+           .. code-block:: console
+
+               conda create -n aiida -c conda-forge aiida-core
+
+    .. tab-item:: source
+
+        Installing ``aiida-core`` directory from source.
+
+        #. Install `git <https://www.git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
+        #. Clone the repository from Github
+
+           .. code-block:: console
+
+               git clone https://github.com/aiidateam/aiida-core
+
+        #. Install `pip <https://pip.pypa.io/en/stable/installation/>`_.
+        #. Install ``aiida-core``:
+
+           .. code-block:: console
+
+               cd aiida-core
+               pip install -e .
+
+           The ``-e`` flag installs the package in editable mode which is recommended for development.
+           Any changes made to the source files are automatically picked up by the installation.
+
+
+.. _installation:guide-complete:python-package:optional-requirements:
+
+Optional requirements
+---------------------
+
+The ``aiida-core`` Python package defines a number of optional requirements, subdivided in the following categories:
+
+* ``atomic_tools`` : Requirements to deal with atomic data and structures
+* ``docs`` : Requirements to build the documentation
+* ``notebook`` : Requirements to run AiiDA in Jupyter notebooks
+* ``pre-commit`` :  Requirements to automatically format and lint source code for development
+* ``rest`` : Requirements to run the REST API
+* ``ssh_kerberos`` : Requirements for enabling SSH authentication through Kerberos
+* ``tests`` : Requirements to run the test suite
+* ``tui`` : Requirements to provide a textual user interface (TUI)
+
+These optional requirements can be installed using pip by adding them as comma separated list, for example:
+
+.. code-block:: console
+
+    pip install aiida-core[atomic_tools,docs]
+
+
+.. _installation:guide-complete:create-profile:
+
+
+Create a profile
+================
+
+After the ``aiida-core`` package is installed, a profile needs to be created.
+A profile defines where the data generated by AiiDA is to be stored.
+The data storage can be customized through plugin and so the required configuration changes based on the selected storage plugin.
+
+To create a new profile, run:
+
+.. code-block:: console
+
+    verdi profile setup <storage_entry_point>
+
+where ``<storage_entry_point>`` is the entry point name of the storage plugin selected for the profile.
+To list the available storage plugins, run:
+
+.. code-block:: console
+
+    verdi plugin list aiida.storage
```

Comment:
It is hidden on purpose. I thought I implemented it such that the command is not even there, but it appears it is just hiding it. I should fix that because it makes no sense to create a `core.sqlite_temp` profile. It was intended to be used only in memory with a context manager and all data is automatically removed when the context manager (or interpreter) is closed.